packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -136,6 +145,8 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
           inviteLink={props.callControlProps.callInvitationURL}
         />
       );
+    } else if (adapter && callStatus === 'LocalHold') {
```

Comment:
not saying its the right thing but just asking if it was evaluated whether the OnHold page should be a CompositePage state?

Reply:
thanks for pointing this out @JamesBurnside 
We didn't evaluate this approach.
My recommendation would also be to make this a composite state instead of doing it in Call Arrangement.
Feel free to reach out in person and discuss

Reply:
Difficulty here is I've realized we've worked ourselves into a corner where this is kinda a breaking change to add a new page in the CompositePage state?
Like if someone is doing `if (state.page !== 'Call') { ... }` they'll be broken and in a way that's hard to know... Not sure the best thing here